lib/upgrade_lts_contract.py

```diff
@@ -34,14 +34,17 @@
     "16.04": "xenial",
     "18.04": "bionic",
     "20.04": "focal",
-    "20.10": "groovy",
+    "21.10": "impish",
+    "22.04": "jammy",
 }
 
 current_codename_to_past_codename = {
     "xenial": "trusty",
     "bionic": "xenial",
     "focal": "bionic",
     "groovy": "focal",
+    "impish": "focal",
+    "jammy": "focal",
```

Comment:
We should leave a comment here explaining that this works even when upgrading from impish -> jammy because there won't be anything enabled on impish anyway.

The output of behave-upgrade-21.10 looks good.

I wonder if we should adjust behave-upgrade-20.04 to upgrade to jammy? and test that too

Reply:
> I wonder if we should adjust behave-upgrade-20.04 to upgrade to jammy? and test that too

I just tried and it doesn't look like this is allowed yet AFAICT

Reply:
I think I manager to solve this, let me know what you think